_collections/_section_release_notes2021/11.6.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Version 11.6
+position: 60
+---
+Release notes
+
+##### New Functionality
+* New property added <b>customCashTreatment</b> as a main toggle to display some features related to the Cash Special Offers
```

Comment:
New property added <b>customCashTreatment</b> as the main toggle to display enhanced cash voucher merchandising.